.circleci/config.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       - run:
           name: Lint
           working_directory: /socorro
-          command: docker run local/socorro_webapp ./docker/run_lint.sh
+          command: docker run local/socorro_app shell ./docker/run_lint.sh
```

Comment:
The `local/socorro_app` image has an entrypoint now and that runs a script that takes a "service" as the first argument. `shell` is a "service" that runs whatever the other arguments are. So in this case it runs the `./docker/run_lint.sh` script.